src/modules/auth/auth.service.ts

```diff
@@ -4,14 +4,22 @@ import {
   Inject,
   Injectable,
   BadRequestException,
+  Body,
 } from '@nestjs/common';
 import * as bcrypt from 'bcrypt';
 import { JwtService } from '@nestjs/jwt';
+import { OAuth2Client } from 'google-auth-library';
 import { UserService } from '@/modules/user/user.service';
 import CreateUserDto from '@/modules/user/dto/create-user.dto';
 import SignInDto from '@/modules/auth/dto/signIn.dto';
 import TokenDto from '@/modules/auth/dto/token.dto';
 import AuthResponseDto from '@/modules/auth/dto/auth-response.dto';
+import { CredentialDto } from '@/modules/auth/googleauth/dto/credential';
+
+const client = new OAuth2Client(
+  process.env.GOOGLE_CLIENT_ID,
```

Comment:
talk to Sergi, he uses next tool for env, really great approach, as he start from good code style, we will try to follow the same

Reply:
fixed

Reply:
I dont see changes